### PR TITLE
Add base sepolia

### PR DESCRIPTION
**Pull request by mdehoog**
**Description**
Follow up to github.com/ethereum-optimism/optimism/pull/7435 that adds Base sepolia. The configuration can be verified in https://github.com/base-org/contract-deployments/edit/main/sepolia/2023-09-26-deploy/deployed/addresses.json and https://github.com/base-org/node/blob/main/sepolia/rollup.json.

Generated bytecodes and genesis using:
```
go run ./op-chain-ops/cmd/registry-data --l2-genesis=../base-node/sepolia/genesis-l2.json --bytecodes-dir=../superchain-registry/superchain/extra/bytecodes --output=../superchain-registry/superchain/extra/genesis/sepolia/base.json.gz
```

### CODE REVIEW

File: superchain/configs/sepolia/base.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Base
+chain_id: 84532
+public_rpc: https://sepolia.base.org
+sequencer_rpc: https://sepolia-sequencer.base.org
+explorer: https://sepolia.basescan.org
```
Comment (mdehoog) [inline superchain/configs/sepolia/base.yaml:5]: coming soon

File: superchain/configs/sepolia/base.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Base
+chain_id: 84532
+public_rpc: https://sepolia.base.org
+sequencer_rpc: https://sepolia-sequencer.base.org
+explorer: https://sepolia-explorer.base.org
```
Comment (mdehoog) [inline superchain/configs/sepolia/base.yaml:5]: coming soon

**Comment by roninjin10** [comment]
@mdehoog just to share, a low priority TODO is to make the indexer use this golang package as it's source of truth so we don't have to do 2 prs for stuff like this

**Comment by protolambda** [comment]
Merged in `main` to try and get the new CI running

**Pull request merged**